src/nvim/main.c

```diff
@@ -215,10 +220,28 @@ void early_init(void)
 
 #ifdef MAKE_LIB
 int nvim_main(int argc, char **argv)
+#elif defined WIN32
+// don't use codepage encoded arguments. see #7060
+int wmain(int argc, wchar_t **argv_w)
```

Comment:
oh, probably not. We simply expect the caller to have encoded the arguments correctly when it calls `nvim_main()`.